tests/integration/nmstatectl_test.py

```diff
@@ -48,10 +48,10 @@
             "ipv6": {
                 "enabled": false
             },
+            "mtu": 65536,
```

Comment:
It seems that you should rather change the order of how mtu/lldp information are inserted instead of adjusting the state. The old format was name, type, state and then sorted. IMHO sorted makes it easiest for users to find a specific value so it would make sense to keep the sorted order.

Reply:
Mixing sorted order with special treatment is confusing and bad on maintenance.
That would require a lot more special handling case by case(like OVS bond name, bridge port name, etc) which I dislike.
It will bite us again when special order is required.

Reply:
The underlying issue is about allowing a special order like showing the OVS bond name first. How is this different from a request to  show lldp before mtu? I am not saying that you need to sort the attributes in prettystate but that if you change this to follow the insertion order that instead of adjusting the test case, the code should be adjusted to insert lldp information before mtu.

Reply:
I don't get it. The order of hashmap/dict is not a part of API. And it's reasonable to group MTU with other ethernet information other than after LLDP information.

Reply:
IMHO the output of nmstatectl is targeted for human users, therefore the order is important to allow humans to find the necessary information fast. In my experience, when I for example would like to look for the mtu of an interface, I would look through it alphabetically. Probably start at some random point in the middle and then if the letter is after "m", I would go up or if it is before "m", I would look down.

Also, the referenced bug reports are about the user experience (having the member names first for bonds/teams), therefore to really solve those bugs, this needs to be tested (and if it is not the case, the properties need to be added in a different order). Otherwise, the original bugs won't be fixed.